tests/test-amp-image-dimension-extractor.php

```diff
@@ -2,8 +2,8 @@
 
 define( 'AMP_IMG_DIMENSION_TEST_VALID_FILE', dirname( __FILE__ ) . '/assets/wordpress-logo.png' );
 define( 'AMP_IMG_DIMENSION_TEST_INVALID_FILE', dirname( __FILE__ ) . '/assets/not-exists.png' );
-define( 'IMG_350', 'https://i0.wp.com/placehold.it/350x150.png');
-define( 'IMG_1024', 'https://i0.wp.com/placehold.it/1024x768.png');
+define( 'IMG_350', 'http://i0.wp.com/placehold.it/350x150.png');
+define( 'IMG_1024', 'http://i0.wp.com/placehold.it/1024x768.png');
```

Comment:
I finally debugged why Travis CI was failing on the PHP 5.2 tests with the changes I tried to push this morning. Sure enough, it was because I'd changed the test image urls back to https, and fopen() (used by the original FastImage library) didn't like it.

"AMP_Image_Dimension_Extractor__By_Downloading__Test::test__multiple_valid_image_files
fopen(): Unable to find the wrapper "https" - did you forget to enable it when you configured PHP?"

It looks like this is happening because Travis' PHP 5.2 environment/installation doesn't load the php_openssl extension (I didn't make an explicit deploy just to confirm this, but that's what the sources I'm reading about the https wrapper error are suggesting). 

I've reverted the test urls to http so that the tests pass in Travis' PHP 5.2 environment.